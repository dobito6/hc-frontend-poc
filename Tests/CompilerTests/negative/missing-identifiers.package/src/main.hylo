fun f(a b, , d: true) {}
