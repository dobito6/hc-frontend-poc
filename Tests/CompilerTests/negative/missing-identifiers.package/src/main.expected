negative/missing-identifiers.package/src/main.hylo:1.11: error: expected parameter declaration
fun f(a b, , d: true) {}
          ^
